extra/onnx_ops.py

```diff
@@ -47,10 +47,7 @@ def Constant(sparse_value:Tensor|None=None, value:Tensor|None=None, value_float:
 
 def HardSigmoid(x:Tensor, alpha:float=0.2, beta:float=0.5): return (alpha*x + beta).clip(0, 1)
 def Gelu(x:Tensor, approximate:str|None=None): return x.gelu() if approximate == "tanh" else 0.5 * x * (1 + (x/math.sqrt(2)).erf())
-# TODO: fix this
-def PRelu(X:Tensor, slope:Tensor):
-  slope = slope[0] if slope.shape[-1] != X.shape[-1] else slope # HACK OnnxBackendPyTorchConvertedModelTest HAS WEIRD SLOPE WHERE IT'S [0.25, 0.25, 0.25] FOR ANY X.SHAPE
-  return (X > 0).where(X, X * slope)
+def PRelu(X:Tensor, slope:Tensor): return (X > 0).where(X, X * slope)
```

Comment:
Why add prelu to Tensor if it can't be used here?

Reply:
I'm still debating on whether or not to add this to tensor...
My attempt to add it with shared behavior was way too hacky.